content/products/desktop/index.html

```diff
@@ -0,0 +1,109 @@
+---
+title: "LocalStack Desktop"
+description: "Get a desktop experience and work with your local LocalStack instance via the UI."
+lead: "LocalStack Desktop, a fully-integrated desktop experience for LocalStack. Start using LocalStack, without the
+fuss."
+date: 2020-08-27T19:25:12+02:00
+lastmod: 2020-08-27T19:25:12+02:00
+draft: false
+images: []
+aliases: ["/[products/cockpit/]"]
+---
+<section class="section section-sm bg-gradient-dark ">
+    <div class="position-absolute w-100 section pb-7 pt-7 d-none d-md-block">
+        <div class="container">
+        <div class="row justify-content-center align-items-center">
+            <div class="col-3">
+            <img
+                class="w-100"
+                style="transform: scale(1.75); opacity: .9;"
+                src="/images/heroes/cloud.svg"
+            />
+            </div>
+            <div class="col-1">
+            </div>
+            <div class="col-2"></div>
+            <div class="col-1">
+            </div>
+            <div class="col-3">
+            <img
+                class="w-100"
+                style="transform: scaleX(-1) scale(1.2)"
+                src="/images/heroes/cloud.svg"
+            />
+            </div>
+        </div>
+        </div>
+    </div>
+    <div class="container pb-7 pt-7 mb-4">
+        <div class="row justify-content-center align-items-center text-white">
+            <h1 class="text-center m-0 pt-5">LocalStack Desktop</h1>
+            <p class="text-center mt-4" style="font-size: 1.5rem">
+                LocalStack Desktop, a fully-integrated desktop experience for LocalStack.
+                <br/>
+                Start using LocalStack - without the fuss
+            </p>
+        </div>
+        <div class="row justify-content-center text-center">
+            <div class="col-12 d-flex justify-content-center gap-3">
+                <a href="https://docs.localstack.cloud/user-guide/tools/localstack-desktop/" style="border: 1px solid white" class="btn btn-second text-white btn-lg">
+                    LEARN MORE
+                </a>
+                <a href="https://app.localstack.cloud/desktop-download/" class="btn btn-primary btn-tertiary btn-lg">
```

Comment:
I would leave it as just /download since that's usually the pattern that people follow
https://www.whatsapp.com/download
https://discord.com/download
https://slack.com/intl/en-gb/downloads/windows

It's the desktop version of the service so I think desktop is superfluos

Reply:
Changed accordingly